### PR TITLE
Stop myFT initialisation console errors for anonymous users

### PR DESCRIPTION
**Pull request by Lrnk**
It's a bit dirty identifying an error from its message. I suppose ideally we'd have the session cookie error be it's own error type and check the error name.
As anonymous behaviour in myFT is currently a bit up in the air, me and @adgad decided it's worth just doing this for now.

### CODE REVIEW

File: src/js-setup.js [hunk 1]
```diff
@@ -94,7 +94,17 @@ JsSetup.prototype.init = function (opts) {
...
+
+			myFtClient.init(myFtOpts)
+				.catch(function (err) {
+
+					if(err.message === 'No session cookie found') {
+						//if user is anonymous, swallow this error to avoid annoying console logs
+						return;
+					}
+
+					throw err;
```
Comment (matthew-andrews) [inline src/js-setup.js]: what do you expect will happen to this error at this point?

Comment (Lrnk) [inline src/js-setup.js]: My understanding is that if it's the session cookie error, it'll get swallowed, but if it's any other error it'll get thrown within the promise like before, as if the catch block was never there (a bit of fiddling: http://www.es6fiddle.net/ielbjhq1/)

**Comment by aintgoin2goa** [comment]
:+1:

File: src/js-setup.js (hunk 1 above)
Comment (matthew-andrews) [inline src/js-setup.js]: ok so long as you don't expect them to go to sentry :+1: :)

**Pull request merged**